### PR TITLE
Support new AtoM API key, refs #10126

### PR DESCRIPTION
**Pull request by sevein**
Redmine: https://projects.artefactual.com/issues/10126

### CODE REVIEW

File: src/dashboard/src/components/helpers.py [hunk 1]
```diff
@@ -219,20 +219,17 @@ def get_atom_levels_of_description(clear=True):
     :param bool clear: When True, deletes all existing levels of description from the Archivematica database before fetching; otherwise, the fetched levels of description will be appended to the already-stored values.
     :raises AtomError: if no AtoM URL or authentication credentials are defined in the settings, or if the levels of description cannot be fetched for another reason
     """
-    url = get_setting('dip_upload_atom_url')
-    if not url:
+    url = get_setting('dip_upload_atom_url', None)
+    if url is None:
```
Comment (Hwesta) [inline src/dashboard/src/components/helpers.py]: Is an empty string a valid AtoM URL?  If not, leaving it as `if not url` may be better.

**Comment by Hwesta** [comment]
:+1: Looks good to me.

**Comment by sevein** [comment]
@Hwesta sorry for the inconveniences, can you review again? I've made some important changes.

**Comment by sevein** [comment]
AtoM 2.3 is finally not going to deprecate basic authentication ([#10135](https://projects.artefactual.com/issues/10135)) so I don't think the work done in this pull request is required anymore.

**Pull request closed without merging**